doc/chromebooks/preparing-chromebook.md

```diff
@@ -0,0 +1,171 @@
+
+after doing this the device should reboot to this screen
+![dev-menu](./assets/boot-menu/old-tablets/dev-menu.png)
+
+you can wait 30 secs for device to continue automatically or go to developer options
+![bootmenu](./assets/boot-menu/old-tablets/boot-menu.png)
+and select to boot from internal disk
+
+</details>
+
+<details>
+<summary>on newer devices</summary>
+
+go to advanced options
+![rev-menu](./assets/boot-menu/newer-chromebook/rev-menu.webp)
+
+and enable developer mode
+
+![adv-options](./assets/boot-menu/newer-chromebook/adv-opt.webp)
+
+</details>
+
+after your devices finishes you will be able to enter tty from chrome os by pressing ```[ctrl]``` ```[alt]``` ```[->]``` or ```[ctrl]``` ```[alt]``` ```[refresh]``` or ```[ctrl]``` ```[alt]``` ```[f2]```
+
+_Note. on the first boot screen there will be a button to enable debug mode, **don't do that** unless you know what you are doing since it's not needed to access the tty and may make doing it more difficult_
+
+# Preparing usb medium
+
+**Important. simply copying image onto an usb is not a correct way of doing it so please don't do it and then open issues about it not working**
+
+## Flashing on Chromebook or Linux
+download the image
+
+using your browser, you can find image for ur specyfic device [here]()
+
+find the name.img.gz (it can be skipped if you already know file location)
+```
+find / -name *.img.gz 2> /dev/null
+```
+_Note. 2> /dev/null is for avoiding throwing useless errors and can be removed_
+
+cd to the directory
+```
+cd path/to/dir
+```
+
+unpack the image
+```
+gunzip name.img.gz
+```
+find your usb device
+```
+lsblk
+```
+_output:_
+```
+luk@chluk /mnt $ lsblk
+NAME         MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
+mtdblock0     31:0    0     8M  0 disk 
+sda      179:0    0 116,5G  0 disk <-- your usb
+├─sda1  179:1    0    32M  0 part <-- partition
+├─sda2  179:2    0    32M  0 part 
+├─sda3  179:3    0   512M  0 part /boot
+├─sda4  179:4    0 108,5G  0 part /
+└─sda5  179:5    0   7,5G  0 part [SWAP]
+```
+_Tip. just find a device with same size as ur usb_
+
+_Note. your partitions might be diffrent_
+
+flash the image
+```
+sudo dd status=progress if=name.img of=/dev/sda
+```
+_Note. replace "sda" with you usb name from the step above_
+
+usb should be ready to go 🎉
+
+_Note. if there is any problem with any command just add sudo before it_
+
+## Flashing on any other system
+
+for simplicity just use [raspberry pi imager](https://www.raspberrypi.com/software/)
+![rpi-imager](./assets/rpi-imager.png)
+
+if you use diffrent software you are on your own, it should work but **do not create and issue about it not working**
+
+_Note. for most you will need to unpack name.img.gz file using archive tool_
+
+# Booting from usb (or sd)
+
+if you are planning to set [gbb flags](./setting_gbb_flags.md) right away you don't have to do this.
```

Comment:
it meant that if you set the gbb flags you don't need to enable usb boot since these force usb boot anyway
the change would make it look like you don't have to set gbb flags which i think is only mentioned as recommended not sth mandatory

Reply:
ok - lets keep it as you had it then